hde-interface/src/HDEInterfaceModule.cpp

```diff
@@ -234,6 +238,13 @@ bool HDEInterfaceModule::updateModule()
         yError() << "HDEInterfaceModule: DoFs mismatch between config file and human joint dynamics";
         return false;
     }
+
+    for(int j=1; j <= hde_controlboard_driver_ptr->number_of_dofs; j++)
+    {
+        hde_controlboard_driver_ptr->joint_positions[j] = hde_controlboard_driver_ptr->joint_positions_rad[j]*(180/M_PI);
+        hde_controlboard_driver_ptr->joint_velocities[j] = hde_controlboard_driver_ptr->joint_velocities_rad[j]*(180/M_PI);
+        hde_controlboard_driver_ptr->joint_accelerations[j] = hde_controlboard_driver_ptr->joint_accelerations_rad[j]*(180/M_PI);
```

Comment:
`M_PI` is not a standard macro, after the demo remember to make sure that this code compiles under Windows.

Reply:
@traversaro Ok :+1: